packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/components/termsOfUse/translations/Messages_fr_FR.json

```diff
@@ -2,6 +2,7 @@
   "dedicatedCloud_datacenter_backup_terms_of_use_description1_new": "Êtes-vous sûr(e) de vouloir activer la sauvegarde dans le datacenter {{datacenter}} ?",
   "dedicatedCloud_datacenter_backup_terms_of_use_description2_new": "Attention, vous serez facturé(e) selon le nombre et la taille des machines virtuelles sauvegardées.",
   "dedicatedCloud_datacenter_backup_terms_of_use_tariff_url": "Consulter en détail la nouvelle gamme Veeam Backup",
-  "dedicatedCloud_datacenter_backup_terms_of_use_general_conditions": "J’ai lu et j’approuve les conditions générales d’utilisation",
+  "dedicatedCloud_datacenter_backup_terms_of_use_general_conditions_text": "J’ai lu et j’approuve les",
+  "dedicatedCloud_datacenter_backup_terms_of_use_general_conditions_url": "conditions générales d’utilisation",
```

Comment:
It doesn't work for all languages, in German we could have the verb at the end of the sentence.

Reply:
This is addressed by combing them into a single translation